robottelo/ui/locators.py

```diff
@@ -753,6 +757,36 @@ def __iter__(self):
 
 locators = LocatorDict({
 
+    # Bookmarks
+    "bookmark.select_name": (
+        By.XPATH,
+        ("//td[following-sibling::td[text()='%s']]"
+         "/a[contains(@href,'bookmarks')][span[contains(.,'%s')]]")),
```

Comment:
The first part searches for rows with a column which contains correct bookmark controller, the second one - just selects a link to desired bookmark. The thing is there can be few bookmarks with the same controller and bookmarks with the same name but different controller :) So need to check both.